SPEC.md

```diff
@@ -65,21 +65,21 @@ Examples: `minitest`, `mysql2`, `battery-horse`.
 
 The version number of the library (String, required).
 
-- It should follow the [Semantic Versioning](http://semver.org/) format.
+- It should follow the [Semantic Versioning 2.0](http://semver.org/spec/v2.0.0.html) format.
```

Comment:
Please do not target a specific version. Anyone is free to choose which version they want, and a new version will eventually come out, which will require updating the SPEC.